reference/shipping.v2.yml

```diff
@@ -1189,6 +1195,25 @@ paths:
           "shipping_cost": 8
         }
         ```
+        
+        ### Channels
+
+        Example request body: 
+
+        ```json
+        {
+          "name": "Method associated to channels 1,3",
```

Comment:
```suggestion
          "name": "Method associated to channels 1, 3",
```

🍹

Reply:
Thanks mate, updated all 3 instances of this. 🍻